tests_metricflow/validation/test_join_validator.py

```diff
@@ -0,0 +1,212 @@
+from __future__ import annotations
+
+from typing import Mapping
+
+from dbt_semantic_interfaces.references import EntityReference
+from metricflow_semantics.model.semantic_manifest_lookup import SemanticManifestLookup
+
+from metricflow.validation.dataflow_join_validator import JoinDataflowOutputValidator
+from tests_metricflow.fixtures.manifest_fixtures import MetricFlowEngineTestFixture, SemanticManifestSetup
+
+
+def test_natural_entity_instance_set_validation(
```

Comment:
These were all a straight copy apart from the imports and calls to the new validator, correct?

Reply:
Yes.